assopy/migrations/0013_auto_20190714_1457.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
```

Comment:
There is another PR that already does that https://github.com/EuroPython/epcon/pull/1076 so we might need to coordinate how this goes in.

One note for the future: please name the migrations. :)